example/linalg/example_inverse1.f90

```diff
@@ -0,0 +1,22 @@
+! Matrix inversion example: operator interface
+program example_inverse1
```

Comment:
I would propose to give explicit names to the example files such as:
example_inverse_operator, example_inverse_function, example_inverse_subroutine

Reply:
[513d77b](https://github.com/fortran-lang/stdlib/pull/828/commits/513d77bc351d22bd7e676f6963c9a3302a26f9e3)